Assets/HoloToolkit/Input/Scripts/Utilities/AttachToController.cs

```diff
@@ -91,5 +78,14 @@ private void SetChildrenActive(bool isActive)
                 }
             }
         }
+
+        protected void Reset()
+        {
+            // We want the default value of Handedness of Controller finders to be Unknown so it doesn't attach to random object.
+            // But we also want the Editor to start with a useful default, so we set a Left handedness on inspector reset.
+#if UNITY_WSA && UNITY_2017_2_OR_NEWER
+            Handedness = InteractionSourceHandedness.Left;
```

Comment:
Handeness looks like the only part that requires UWP, is that correct?

Reply:
Yep